### DISCUSSION TITLE
Which OBD2 interface box to use with ddt4all ?

### DISCUSSION THREAD

**Discussion by tof-59** [cedricp/ddt4all#1185] [Q&A]
Hi everyone,

I installed the last version of ddt4all dowloaded here and installed also the database, for that it is OK.
I'm actually trying to connect to my car (Renault Mégane 3 from 2011) and it doesn't works..

It remains blocked at step :
"Initialisation ELM a la vitesse 38400...
Tentative d'ouverture du port 192.168.0.10:35000 @ 38400"
(it is in french because i'm french...)

Then if I wait it is trying others speeds but the same results.
Of course, I'm connected by wifi to my box OBD2...

I checked some topics on this page and others people had this problem, and it seems that come from the OBD box interface that is not compatible.
I think that is the case for me because the box works fine with some software I tested on android (EOBD Facile for example).
That is surprising is there are video on youtube (date of 2019) where we can see ddat4all running with my model of box, but it seems that a componant had changed on new model and now it's not compatible...

For info, my box : https://www.amazon.fr/dp/B01NAQ2878?psc=1

**My question is : do you have an updated list or recommandation with OBD2 interface bow that works with ddt4all ??**

Thanks you,

**Comment by matzel82** [comment]
You can buy an OBD Link SX

I have it..it’s work perfectly

**Comment by AndyIreland** [comment]
can you maybe split open your ELM and see if it has the required PIC18F25K80 chip on the main board? 

**According to the requirements:**
Supported diagnostic adapters (so far)
ELM327 USB/BlueTooth/WiFi (Original one with PIC18F25K80, Chinese clone not working)
ObdLink SX

**Comment by tof-59** [comment]
Hi @matzel82, @AndyIreland 

Thanks you for your answers.
On my side I succeeded to solve my problem by the download of an other version of ddt4all : yesterday I was with the 3.0.1 find here (the cedricp github page), then I tried with a version "5.6.0" find on an other github page.
I think it is a fork maybe, at first glance there is less functions than the "original" but it works fine with my interface so it is ok for me.

For the chip of my interface, it is identified in EOBD Facile as a version 2.2 of ELM327, but I didn't open it.

**Reply by enesokyay** [reply]
Elm327 WiFi OBDII & Srvice... https://www.amazon.fr/dp/B0CXW88FFM?

I have this version too. When I'm connected with the phone, I can easily connect and process, but the macOS m1 processor doesn't do that. I set the ip4 manually but still "Initialisation ELM a la vitesse 38400...

Tentative d'ouverture du port remains in 192.168.0.10:35000 @ 38400".

**Reply by ivansnake** [reply]
> Hi @matzel82, @AndyIreland
> 
> Thanks you for your answers. On my side I succeeded to solve my problem by the download of an other version of ddt4all : yesterday I was with the 3.0.1 find here (the cedricp github page), then I tried with a version "5.6.0" find on an other github page. I think it is a fork maybe, at first glance there is less functions than the "original" but it works fine with my interface so it is ok for me.
> 
> For the chip of my interface, it is identified in EOBD Facile as a version 2.2 of ELM327, but I didn't open it.

Salut, j'ai le même problème, pourrais-tu m'envoyer le lien pour avoir la version 5.6.0 j'ai trouvé que la version 5.5.0
Merci
